src/Helper/PaymentMethodFormatter.php

```
@@ -0,0 +1,372 @@
+        }
+
+        return $paymentMethodDisplay;
+    }
+
+    /**
+     * Get localized payment method name
+     */
+    public function getLocalizedPaymentMethodName(string $method, ?string $brand = null): string
+    {
+        $context = \Context::getContext();
+        $langId = (int) $context->language->id;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$langId'. (Unused Code Rules)

(UnusedLocalVariable)